app/src/androidTest/java/ch/epfl/sdp/musiconnect/VisitorProfileTest.java

```diff
@@ -102,14 +102,6 @@ private void testMusician(Musician m) {
         visitorActivityTestRule.finishActivity();
     }
 
-    @Test

```

Comment:
Is there a particular reason to remove this test?

Reply:
yes, it is nearly the same as a test already done by @JulienHu so I removed the duplicate.